lib/active_remote/version.rb

```diff
@@ -1,3 +1,3 @@
 module ActiveRemote
-  VERSION = "6.1.1"
+  VERSION = "7.0.0"
```

Comment:
I prefer to bump the version while releasing to more easily group changes together.